### ISSUE TITLE
Upload fails on 4.0.0

### ISSUE THREAD

**Issue by latica-jp** [kintone/plugin-uploader#264]
On 3.0.6 - 4.0.0, plugin upload fails with following error: 

```
Open https://mydomain.cybozu.com/login?saml=off
Trying to log in...
Navigate to https://mydomain.cybozu.com/k/admin/system/plugin/
Trying to upload ./dist/plugin.zip
エラーが発生しました { TimeoutError: waiting for selector ".ocean-ui-dialog" to be hidden failed: timeout 5000ms exceeded
    at new WaitTask (/Users/isao/Documents/kintone Projects/kintone-plugin-template/node_modules/puppeteer/lib/DOMWorld.js:549:28)
    at DOMWorld._waitForSelectorOrXPath (/Users/isao/Documents/kintone Projects/kintone-plugin-template/node_modules/puppeteer/lib/DOMWorld.js:478:22)
    at DOMWorld.waitForSelector (/Users/isao/Documents/kintone Projects/kintone-plugin-template/node_modules/puppeteer/lib/DOMWorld.js:432:17)
    at Frame.waitForSelector (/Users/isao/Documents/kintone Projects/kintone-plugin-template/node_modules/puppeteer/lib/FrameManager.js:627:47)
    at Frame.<anonymous> (/Users/isao/Documents/kintone Projects/kintone-plugin-template/node_modules/puppeteer/lib/helper.js:112:23)
    at Page.waitForSelector (/Users/isao/Documents/kintone Projects/kintone-plugin-template/node_modules/puppeteer/lib/Page.js:1120:29)
    at /Users/isao/Documents/kintone Projects/kintone-plugin-template/node_modules/@kintone/plugin-uploader/dist/index.js:81:20
    at Generator.next (<anonymous>)
    at fulfilled (/Users/isao/Documents/kintone Projects/kintone-plugin-template/node_modules/@kintone/plugin-uploader/dist/index.js:5:58)
    at process._tickCallback (internal/process/next_tick.js:68:7) name: 'TimeoutError' }
``` 

Besides, 3.0.5 w/ puppeteer 1.19.0 works fine. 

I've confirmed that `elementHandle.uploadFile` doesn't work as expected after 3.0.6, so it seems to be a puppeteer's problem.

## Steps to reproduce

`npx kintone-plugin-uploader --domain mydomain.cybozu.com --username USERNAME --password PASSWORD RELATIVE_PATH_TO_PLUGIN_ZIP`

**Comment by zaki-yama** [comment]
@latica-jp 
Thank you for reporting!
As you have guessed, it seems to be a puppeteer's problem.
The issue is here: https://github.com/puppeteer/puppeteer/issues/5363

Since the issue was already closed, we will wait for the new version of puppeteer to be published.
If the fixed version isn't released in a few days, we'll downgrade puppeteer temporarily.

We'll notify you if there is any progress.

**Comment by koba04** [comment]
@latica-jp
I've pinned puppeteer version v1 to avoid this issue and have released it as v4.0.1.
I'll work on updating `puppeteer` to the latest version later.
Thank you for your contribution!!
